src/dss/remove_notebook.py

```diff
@@ -0,0 +1,34 @@
+from lightkube import Client
+from lightkube.core.exceptions import ApiError
+from lightkube.resources.apps_v1 import Deployment
+from lightkube.resources.core_v1 import Service
+
+from dss.config import DSS_NAMESPACE
+from dss.logger import setup_logger
+
+# Set up logger
+logger = setup_logger("logs/dss.log")
+
+
+def remove_notebook(name: str, lightkube_client: Client) -> None:
+    """
+    Removes a Notebook server from the Kubernetes cluster.
```

Comment:
`Notebook` is a bit abstract - can you have a second paragraph that says specifically what will be removed (the `Deployment` and `Service` of name `name`)

Reply:
added